readthedocs/projects/tasks/utils.py

```diff
@@ -154,6 +161,115 @@ def send_external_build_status(version_type, build_pk, commit, status):
         send_build_status.delay(build_pk, commit, status)
 
 
+class DeprecatedConfigFileSiteNotification(SiteNotification):
+
+    failure_message = (
+        "Your project '{{ object.slug }}' doesn't have a "
+        "<code>.readthedocs.yaml</code> configuration file. "
+        "Configuration files will <strong>soon be required</strong> by projects, "
+        "and will no longer be optional. "
+        '<a href="https://blog.readthedocs.com/migrate-configuration-v2/">Read our blog post to create one</a> '  # noqa
+        "to ensure your project continues building successfully."
+    )
+    failure_level = WARNING_PERSISTENT
+
+
+class DeprecatedConfigFileEmailNotification(Notification):
+
+    app_templates = "projects"
+    name = "deprecated_config_file_used"
+    context_object_name = "project"
+    subject = "Add a configuration file to your project to continue building"
```

Comment:
Considering changing this subject after some weeks 👍🏼

Reply:
This should be something like this:

```suggestion
    subject = "[ACTION REQUIRED] Add a configuration file to your project to continue building"
```

Reply:
I found this subject line in my inbox: `[Netlify] Action required to prevent build failure`. I think we can use something pretty similar here as well.